sklearn/linear_model/tests/test_bayes.py

```diff
@@ -110,6 +111,22 @@ def test_std_bayesian_ridge_ard_with_constant_input():
         assert_array_less(y_std, expected_upper_boundary)
 
 
+def test_regression_issue_10128():
+    # this ARDRegression test throws a `ValueError` on master, commit 5963fd2
+    np.random.seed(752)
```

Comment:
I think that we don't have to rely on a random seed and we can create a deterministic toy example.
We should check the data (specifically the unique value) and create such a matrix as input

Reply:
still haven't succeeded in coming up with a smaller example.

Reply:
Ok I came up with a better example! Will push it later

Reply:
done